web/styles/modal.css

```diff
@@ -86,7 +86,7 @@
     flex-direction: column;
     font-size: 1rem;
     overflow-y: auto;
-    padding: 0 24px;
+    padding: 2px 24px;
```

Comment:
Can you explain this change more, maybe with before/after screenshots? Might be good to just do this in a separate prep PR since this PR thread is already very long.

Reply:
Opened #26500.